src/components/Events/EventCard.js

```diff
@@ -0,0 +1,41 @@
+import { React } from 'react';
+
+import EventItem from './EventItem'
+import './EventCard.css'
+
+function EventCard (props) {
+
+    const data = props.eventData
+    console.log('props data:', data)
+
+    // const currentDate = new Date()
+    // console.log('currentDate:', currentDate)
+
+    // const isPast = (date) => {
+    //    if (date < currentDate) {
+    //     //    return true
+    //     console.log(date < currentDate)
+    //    }
+    // };
+
+    const isResultNull = (result) => {
+        if (result === null) {
```

Comment:
`=== null` is only checking if it's exactly `null`. 
but `== null` is checking if it's also `undefined`. Thus, `== null` is used commonly.